CHANGELOG.md

```diff
@@ -48,6 +48,7 @@
 ### 🧰 Bug fixes 🧰
 
 - Fixed bug where `telemetryInitializer` is not cleaned up when `newService` errors (#6239)
+- Fixed collector attempting startup when `ShutDown` is called before `Run`
```

Comment:
Can you add the PR number in parenthesis here please? 

So just adding `(#6279)` at the end.

Reply:
Oh okay, thanks a lot. I do not know how I missed that